tox.ini

```diff
@@ -7,10 +7,10 @@ whitelist_externals =
 deps =
     -r{toxinidir}/test-requirements.txt
 commands =
-    black diff_cover
-    isort diff_cover
+    black diff_cover tests
+    isort diff_cover tests
     python -m pytest --cov --cov-report=xml tests
     git fetch origin main:refs/remotes/origin/main
-    diff-cover coverage.xml
-    diff-quality --violation=flake8
-    diff-quality --violation=pylint
+    diff-cover coverage.xml --include-untracked
+    diff-quality --violations flake8 --include-untracked
+    diff-quality --violations pylint --include-untracked
```

Comment:
Im a bit skeptical this will be a great test since in CI there would not be untracked files but I guess it will let us know if the git side breaks hard